templates/rabbitmq.config.erb

```diff
@@ -1,6 +1,9 @@
 % This file managed by Puppet
 % Template Path: <%= @module_name %>/templates/rabbitmq.config
 [
+<%- if @ssl and @ssl_versions -%>
+  {ssl, [{versions, [<%= @ssl_versions.sort.map { |v| "'#{v}'" }.join(', ') %>]}]},
+<%- end -%>
```

Comment:
No, because it doesn't actually do anything functional in my tests.
The docs place it outside the rabbit config scope, so that is where it should be unless the docs are wrong.